__tests__/unit/integration/api.test.js

```diff
@@ -0,0 +1,148 @@
+const { appendToFile } = require("../../../routes/api");
```

Comment:
this folder location seems weird, shouldn't `integration` be a sibling directory of `unit` and not a child directory?